probe_scraper/glean_checks.py

```diff
@@ -142,44 +142,49 @@ def check_for_duplicate_metrics(repositories, metrics_by_repo, emails):
     return found_duplicates
 
 
+FIREFOX_DESKTOP_EXPIRY_LANGUAGE = (
+    "expire in the next version (Firefox {latest_nightly_version})"
+)
+DATE_BASED_EXPIRY_LANGUAGE = "expire in the next {expire_days} days"
 EXPIRED_METRICS_EMAIL_TEMPLATE = """
-The following metrics in {repo_name} will expire in the next {expire_days} days or have already expired.
+Each metric in the following list from {repo_name} will {expiry_language} or has already expired.
```

Comment:
"The following metrics in" reads a little better to me, any reason why you changed the language?

Also, I think ideally we would list out each set of metrics (has expired, soon to expire) separately. It might be worth considering using a jinja template here, which would make it easier to express this type of conditional logic.